source/fundamentals/transactions.txt

```diff
@@ -208,6 +208,15 @@ the Convenient Transaction API:
 To see a fully runnable example that uses this API, see the
 :ref:`node-usage-convenient-txn` usage example.
 
+.. sharedinclude:: dbx/transactions-parallelism.rst
+
+   .. replacement:: driver-specific-content
+
+      If you're using {+mdb-server+} v8.0 or later, you can perform
+      write operations on multiple namespaces within a single transaction by using
+      the ``BulkWrite()`` method. For more information,
```

Comment:
S: looks like the method name is camel case in Node
```suggestion
      the ``bulkWrite()`` method. For more information,
```

Reply:
Ah thanks for catching!